config/application.rb

```diff
@@ -184,11 +184,15 @@ def parse_bool(val, default = false)
     # Default admin password
     config.admin_password_default = ENV['ADMIN_PASSWORD'] || 'administrator'
     # custom parameter
```

Comment:
# Application parameters (https://docs.bigbluebutton.org/admin/customize.html#passing-custom-parameters-to-the-client-on-join)